src/test/java/org/Simplycodestudio/CinemaWorld/TicketBookingApp/domains/RoomDtoTest.groovy

```diff
@@ -0,0 +1,23 @@
+package org.Simplycodestudio.CinemaWorld.TicketBookingApp.domains
+
+
+import spock.lang.Specification
+
+class RoomDtoTest extends Specification {
+
+    def "Test should pass when OutputHashMap has the same value as InputStringMap"() {
+
+        when:
+        String inputStringMap = "{2:[1,2,5,6,9,10,13,14]}"
```

Comment:
Te wszystkie testy sprawdzają to co łatwo się testuje, a nie to co jest ważne :(

Reply:
Zgadzam się. Testowanie zapytań, odpowiedzi i mockowanie takich rzeczy jest dla mnie zupełną nowością (głównie spocku). Nie chciałem wprowadzać testów, których struktury się dopiero uczę.